expression/builtin_string_vec_test.go

```diff
@@ -256,6 +256,9 @@ var vecBuiltinStringCases = map[string][]vecExprBenchCase{
 			},
 		},
 	},
+	ast.Instr: {
+		{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETString, types.ETString}},
+	},
```

Comment:
Could you please add some test cases in which the second argument is the substring of the first argument?

Reply:
`{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETString, types.ETString}, geners: []dataGenerator{&constStrGener{"test,case"}, &constStrGener{"case"}}},

{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETString, types.ETString}, geners: []dataGenerator{&constStrGener{"test,case"}, &constStrGener{"testcase"}}},`
Is it ok?